Description: make contrib/slapd-modules/smbkrb5pwd/Makefile debian-friendly

Author: Tuomas Räsänen <tuomasjjrasanen@tjjr.fi>

--- openldap-2.4.37.orig/contrib/slapd-modules/smbkrb5pwd/Makefile
+++ openldap-2.4.37/contrib/slapd-modules/smbkrb5pwd/Makefile
@@ -13,7 +13,7 @@
 # <http://www.OpenLDAP.org/license.html>.
 
 LDAP_SRC=../../..
-LDAP_BUILD=$(LDAP_SRC)
+LDAP_BUILD=$(LDAP_SRC)/debian/build
 LDAP_INC=-I$(LDAP_BUILD)/include -I$(LDAP_SRC)/include -I$(LDAP_SRC)/servers/slapd
 LDAP_LIB=$(LDAP_BUILD)/libraries/libldap_r/libldap_r.la \
 	$(LDAP_BUILD)/libraries/liblber/liblber.la
@@ -37,11 +37,11 @@ LIBS=$(LDAP_LIB) $(MIT_KRB5_LIB) $(SSL_L
 MIT_KRB5_SRV_LIB=-lkadm5srv_mit
 MIT_KRB5_CLNT_LIB=-lkadm5clnt_mit
 
-prefix=/usr/local
-ldap_subdir=/openldap
+prefix=/usr
+ldap_subdir=/ldap
 
 libdir=$(prefix)/lib
-libexecdir=$(prefix)/libexec
+libexecdir=$(prefix)/lib
 moduledir=$(libexecdir)$(ldap_subdir)
 
 .PHONY: all
